subprojects/common/logger-test-fixtures/src/main/kotlin/com/avito/logger/NoOpLogger.kt

```diff
@@ -1,9 +1,11 @@
 package com.avito.logger
 
-object FakeLogger: Logger {
```

Comment:
We use the term `Fake` usually for stubs in tests (e.g. FakeReport, FakeTestsRunner, ...).
This is why I want to use the term `NoOp` for literally empty implementations.